src/main.rs

```diff
@@ -43,11 +43,15 @@ async fn main() -> fpm::Result<()> {
         if build.is_present("verbose") {
             println!("{}", fpm::debug_env_vars());
         }
+        // Build only non-static files when file is passed
+        let no_static: bool = build.value_of("file").is_some();
+
```

Comment:
Remove this, add this condition in build2 itself

Reply:
Added this condition in build2()